tests/examples-without-external-providers/port-2-ibm2-connection/README.md

```diff
@@ -0,0 +1,349 @@
+# Fabric Port to Fabric IBM 2.0 Service Profile Connection
```

Comment:
@thogarty - As you mentioned in your previous comment, we don't need README.md here